### PR TITLE
Fix bug while writing to parcel

### PR DESCRIPTION
**Pull request by diegoserranoa**
There's a bug when writing to parcel and the object is null.
This PR adds a null check to prevent an exception.

## Test

1. Set a `WootricCustomThankYou`
```java
        WootricCustomThankYou ty = new WootricCustomThankYou();
        ty.setText("Test");
        wootric.setCustomThankYou(ty);
```
2. Let the survey show up, then hit the home button to put the app in background. In master, this should make the app crash. In this branch, the app shouldn't crash and should work normally.

This fixes the issue: https://github.com/Wootric/WootricSDK-Android/issues/79

### CODE REVIEW

**Comment by prabhatjha** [comment]
@diegoserranoa I know it's a critical fix. We missed it probably because we did not test for negative condition. Can you add some tests around it?

File: androidsdk/src/main/java/com/wootric/androidsdk/utils/Utils.java [hunk 1]
```diff
@@ -67,4 +67,12 @@ public static void checkDate(long date) {
...
         }
     }
+
+    public static byte getByteValue(Boolean bool) {
+        if (bool != null) {
+            return bool ? (byte) 1 : (byte) 0;
+        } else {
+            return (byte) 0;
+        }
+    }
```
Comment (hachibu) [inline androidsdk/src/main/java/com/wootric/androidsdk/utils/Utils.java]: Makes sense to me!

**Comment by hachibu** [comment]
I agree with @prabhatjha can you add a small test.

**Comment by diegoserranoa** [comment]
Thanks for the review! Added the test @hachibu @prabhatjha

**Pull request merged**